sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -952,9 +942,35 @@ export class MessageReceiver extends LinkEntity {
           connectionHost: this._context.namespace.config.host,
           delayInSeconds: 15
         };
-        if (!this.wasCloseInitiated) {
-          await retry<void>(config);
-        }
+        await retry<void>(config);
+      } else if (causedByDisconnect) {
+        const state: any = {
+          wasCloseInitiated: this.wasCloseInitiated,
+          receiverError: translatedError,
+          _receiver: this._receiver
```

Comment:
what is the point of logging the entire receiver? Is there anything on it that will help in diagnosis?

Reply:
You caught me...I copied the logs we had before this change for the exceptional case without really thinking if they were helpful. Will remove.

Reply:
I believe it mattered before as it contained info on the `receiver.isItselfClosed()`